helper/config/deprecated_options.go

```diff
@@ -2,43 +2,20 @@
 
 package config
 
-var DeprecatedOptions = []string{
-	"iso_md5",
-	"guest_additions_attach",
-	"headless",
-	"parallels_tools_host_path",
-	"guest_os_distribution",
-	"ssh_key_path",
-	"ssh_disable_agent",
-	"access_key",
-	"filename",
-	"shutdown_behaviour",
-	"enhanced_networking",
-	"ssh_private_ip",
-	"temporary_security_group_source_cidr",
-	"login_email",
-	"tag",
-	"vhd_temp_path",
-	"clone_from_vmxc_path",
-	"cpu",
-	"ram_size",
-	"clean_image_name",
-	"clean_ami_name",
-	"spot_price_auto_product",
-	"galaxycommand",
-	"ssh_host_port_min",
-	"ssh_host_port_max",
-	"ssh_skip_nat_mapping",
-	"ssh_wait_timeout",
-	"network_card",
-	"network",
-	"networkCard",
-	"disk_size",
-	"disk_thin_provisioned",
-	"disk_eagerly_scrub",
-	"iso_checksum_url",
-	"iso_checksum_type",
-	"ssh_host_port_max",
-	"ssh_host_port_min",
-	"exlude_from_latest",
+var DeprecatedOptions = map[string][]string{
+	"*":                                   []string{"iso_md5", "ssh_key_path", "ssh_disable_agent", "ssh_host_port_min", "ssh_host_port_max", "ssh_skip_nat_mapping", "ssh_wait_timeout", "iso_checksum_url", "iso_checksum_type"},
```

Comment:
While testing this out with a few different configs I got thrown for a loop because I expected a warning for `ssh_wait_timeout` but because it is not truly deprecated as in the option is merely undocumented and not remove no warning was disabled :facepalm: .